Makefile

```diff
@@ -35,6 +35,11 @@ quiet-test:
 	@opa test .
 	@opa test . --threshold 100 >/dev/null 2>&1
 
+# Install https://github.com/eradman/entr then run this in a
+# separate terminal or split window while hacking
+live-test:
+	@git ls-files '*.rego' | entr -c $(MAKE) --no-print-directory quiet-test
```

Comment:
`git ls-files '*.rego'` will miss new files that have not yet been added to git. We probably want `find . -name '*.rego'`

Reply:
good point.

Reply:
`git ls-files -c -o` should do it.

Reply:
..and a loop. `entr -d` means quit if any files appeared or disappeared.